dashboard/src/components/Checkbox/IndeterminateCheckbox.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useRef, type HTMLProps } from 'react';
+
+import { cn } from '@/lib/utils';
+
+export function IndeterminateCheckbox({
+  indeterminate,
```

Comment:
if you set `indeterminate = false`, you can remove `if` statemente in line 13.

Reply:
Makes sense, thanks